.circleci/config.yml

```diff
@@ -64,6 +64,25 @@ jobs:
       - *docker-login
       - *build-release
 
+  deploy-macos:
+    macos:
+        xcode: "10.0.0"
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: bootstrap setup
+          command: |
+            curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```

Comment:
Does Circle CI have os x machines with conda already installed? Perhaps we could cache these dependencies?

Reply:
Don't think so. Will look into caching.